### ISSUE TITLE
Date entry for workshop format prompt

### ISSUE THREAD

**Issue by gerboland** [PythonIreland/broadgauge#3]
The date entry field on the new workshop page is in a awkward format with no prompt.

Current format required is 2015-02-13

We would like a prompt within the field for users to quickly understand the date entry format required.

DOD: Date entry field contains a sample of date format to be entered.

**Comment by andrewmccarthy** [comment]
There are a number of possibilities:
- HTML5 now has a date input format:
  
  ``` html
  <input type="date" name="date">
  ```
  
  however this is not yet universally supported on browsers.
- There are datepickers for bootstrap
- Just add the date format YYYY-MM-DD to the form label (easy)

**Comment by andrewmccarthy** [comment]
Just a note to say the HTML already uses the date input field:

``` html
        <label for="date"><label for="date">Preferred Date</label> <span class="small">Provide date in format YYYY-MM-DD</span></label>
        <input class="form-control" id="date" name="date" type="date" value="2015-06-20">
```

**Comment by andrewmccarthy** [comment]
Screenshot of the proposed fix as it appears in Firefox 38.0 on Linux:
![date_prompt](https://cloud.githubusercontent.com/assets/6887733/8267659/58eb948a-1762-11e5-9e35-cd6a78e279cd.png)
